src/libs/TransactionUtils/index.ts

```diff
@@ -1218,6 +1231,30 @@ function buildTransactionsMergeParams(reviewDuplicates: OnyxEntry<ReviewDuplicat
     };
 }
 
+/**
+ * Return the sorted list transactions of an iou report
+ */
+function getAllSortedTransactions(iouReportID: string): Array<OnyxEntry<Transaction>> {
+    // We need sort all transactions by sorting the parent report actions because `created` of the transaction only has format `YYYY-MM-DD` which can cause the wrong sorting
```

Comment:
> // We need sort all transactions by sorting the parent report actions because `created` of the transaction only has format `YYYY-MM-DD` which can cause the wrong sorting

@Beamanator Currently, we're going to the first expense. I think I need to update the comment a bit to 

```
// We need to sort all transactions by sorting the parent report actions because `created` of the transaction doesn't mean the created time of the transaction.
```

Reply:
Ok well from my testing in OldDot, it honestly doesn't seem consistent, which order the category approvers show up when there's multiple 😅  So I'm discussing internally to figure out what we'll want to do in NewDot, as an expected order

Reply:
Ok Updated sort order is here: https://github.com/Expensify/App/issues/52458#issuecomment-2504379629

Reply:
Still investigate the sort order.

Reply:
@ntdiary Updated the sort order.